src/corona_hakab_model/katriel_validation.py

```diff
@@ -0,0 +1,196 @@
+import matplotlib.pyplot as plt
```

Comment:
Please put this file in a folder with valid name (corona_hakab_model is stuffed with more files than my middle drawer)

Reply:
I did it in the beginning, but when you run it from another folder all of the relatives CSV export paths break :(
Any idea on how to fix it?

Reply:
project_structure.py?